### ISSUE TITLE
(mkvtoolnix) Should this be a meta package?

### ISSUE THREAD

**Issue by gep13** [chocolatey/chocolatey-coreteampackages#1016]
We have this package:

https://chocolatey.org/packages/mkvtoolnix

And another maintainer has published this package:

https://chocolatey.org/packages/mkvtoolnix.portable

Should we look to bring the portable version under the core team packages repo, and create a mkvtoolnix.install?

What are your thoughts @echizenryoma

**Comment by RedBaron2** [comment]
I'm going to throw my two cents in here. It appears that the packages are similar to the `CrystalDiskInfo` packages. These packages have a stand alone version (install-able), and a portable version.
I think the discussion should be really about Chocolatey having a switch for choosing `--portable` much like the `--pre` is for alpha/beta/pre releases.
Just my two cents.

**Comment by gep13** [comment]
@RedBaron2 I think that is a good feature request for Chocolatey, but I don't really see how it is relevant to the question that I was asking.  Did you want to create that as a feature request on the Choco GitHub repo?

**Comment by RedBaron2** [comment]
@gep13 I can do that.
Would this be like my meta for the win10mct?

**Comment by gep13** [comment]
>@RedBaron2 said...
Would this be like my meta for the win10mct?

sorry, not sure I follow what you mean.

**Comment by RedBaron2** [comment]
The win10mct is the aka for windows10-media-creation-tool
The windows10-media-creation-tool points to win10mct just like the javaruntime points to jre8

**Comment by gep13** [comment]
Ah, I see, you are essentially talking about an alias for a package.

In this instance, I was talking about the trio of packages, i.e. 

* mkvtoolnix
* mkvtoolnix.portable
* mkvtoolnix.install

One of these currently resides in this repository, and another lives in @echizenryoma repository, and the other doesn't yet exist.  My suggestion was to bring them all under one repository, so that they update/publish at the same time.

This concept is "similar" to aliasing a package name.

**Comment by majkinetor** [comment]
There are number of other packages like this. I don't see much of the benefit and it complicates stuff for development given the only reason we do so is  discoverability which IMO doesn't improve if main package = install and portable is named like before (like this one, but yesterday I worked on tagscanner with the same property). Not to mention compliction when all packages are not born the same day.

Anyway, the process could be streamlined in AU using extra user script/function and streams, as packages use different installers and uninstallers, although a single one can be created for both (which I did on 1 I think) but it might confuse people who read source code of the package.

Its simply not maintanable to have 3 packages sharing almost everything. The entire concept of meta,install, portable should be IMO abandoned in this form in at least development (3 packages). /cc @AdmiringWorm
